keyboards/keebio/iris/keymaps/merrittlj/README.MD

```diff
@@ -0,0 +1,15 @@
+![merrittlj keymap images](https://imgur.com/a/7AL7Yxq)
```

Comment:
Unfortunately you'll have to get a link for every image in this link, including the added `h`, and add them in sequence.

Reply:
Is stitching all of the layer images together into one image, an option?

Reply:
> What do you mean exactly by "the added h"?
Taken from PR checklist:
- a picture about the keyboard and preferably about the PCB, too
    - images are not to be placed in the qmk_firmware repository
    - images should be uploaded to an external image hosting service, such as imgur.
    - if imgur is used, images should be resized appropriately: append “h” to the image url i.e. https://i.imgur.com/vqgE7Ok.jpg becomes https://i.imgur.com/vqgE7Okh.jpg
    - image links should link directly to the image, not a “preview” – i.e. https://imgur.com/vqgE7Ok should be https://i.imgur.com/vqgE7Okh.jpg when using imgur

Reply:
> Is stitching all of the layer images together into one image, an option?

Not from imgur directly, you'd need to stitch them together and upload the result to imgur.